rpc: update transactions page (#1816)
near/docs: docs/5.api/rpc/transactions.md

- method: `send_tx`
- params: 
  - `signed_tx_base64`: SignedTransaction encoded in base64

>   - [Optional] `wait_until`: the required minimal execution level. Read more [here](/api/rpc/transactions#tx-status-result). The default value is `EXECUTED_OPTIMISTIC`.

Author: Please check if the links work, I have no idea how to check this locally, I just copy pasted the link format somewhere

>   - [Optional] `wait_until`: the required minimal execution level. Read more [here](/api/rpc/transactions#tx-status-result). The default value is `EXECUTED_OPTIMISTIC`.

Reviewer: ```suggestion
  - [Optional] `wait_until`: the required minimal execution level. [Read more here](#tx-status-result). The default value is `EXECUTED_OPTIMISTIC`.
```

- method: `tx`
- params:
  - `tx_hash` _(see [NearBlocks Explorer](https://testnet.nearblocks.io) for a valid transaction hash)_
  - `sender_account_id` _(used to determine which shard to query for transaction)_
  - [Optional] `wait_until`: the required minimal execution level. Read more [here](/api/rpc/transactions#tx-status-result). The default value is `EXECUTED_OPTIMISTIC`.

Transaction status request with `wait_until != NONE` will wait until the transaction appears in the chain.
If the transaction does not exist, the method will wait until the timeout is reached.

> If you just need to check whether the transaction exists, use `wait_until = NONE`, it will return the response immediately.

Author: Please proofread these 3 lines and rephrase them if you know how.

> Transaction status request with `wait_until != NONE` will wait until the transaction appears in the chain.
> If the transaction does not exist, the method will wait until the timeout is reached.
> If you just need to check whether the transaction exists, use `wait_until = NONE`, it will return the response immediately.

Reviewer: ```suggestion
A Transaction status request with `wait_until != NONE` will wait until the transaction appears on the blockchain.
If the transaction does not exist, the method will wait until the timeout is reached.
If you only need to check whether the transaction exists, use `wait_until = NONE`, it will return the response immediately.
```

  "jsonrpc": "2.0",
  "id": "dontcare",
  "method": "tx",
  "params": ["6zgh2u9DqHHiXzdy9ouTP7oGky2T4nugqzqt9wJZwNFm", "sender.testnet"]
  "params": {
    "tx_hash": "6zgh2u9DqHHiXzdy9ouTP7oGky2T4nugqzqt9wJZwNFm",
    "sender_account_id": "sender.testnet",
    "wait_until": "EXECUTED"
  }
}
```

</TabItem>
<TabItem value="🌐 JavaScript" label="JavaScript">

```js
const response = await near.connection.provider.txStatus(
  "6zgh2u9DqHHiXzdy9ouTP7oGky2T4nugqzqt9wJZwNFm",
  "sender.testnet"
);
```

> 

Author: I decided to drop js examples because it's hard to maintain them, we update the libraries too frequently
The examples in the libraries should be enough

- method: `EXPERIMENTAL_tx_status`
- params:
  - `tx_hash` _(see [NearBlocks Explorer](https://testnet.nearblocks.io) for a valid transaction hash)_
  - `sender_account_id` _(used to determine which shard to query for transaction)_
  - [Optional] `wait_until`: the required minimal execution level. Read more [here](/api/rpc/transactions#tx-status-result). The default value is `EXECUTED_OPTIMISTIC`.

Transaction status request with `wait_until != NONE` will wait until the transaction appears in the chain.
If the transaction does not exist, the method will wait until the timeout is reached.

> If you just need to check whether the transaction exists, use `wait_until = NONE`, it will return the response immediately.

Author: [same text]

> Transaction status request with `wait_until != NONE` will wait until the transaction appears in the chain.
> If the transaction does not exist, the method will wait until the timeout is reached.
> If you just need to check whether the transaction exists, use `wait_until = NONE`, it will return the response immediately.

Reviewer: ```suggestion
A Transaction status request with `wait_until != NONE` will wait until the transaction appears on the blockchain.
If the transaction does not exist, the method will wait until the timeout is reached.
If you only need to check whether the transaction exists, use `wait_until = NONE`, it will return the response immediately.
```
